### PR TITLE
Switch affiliations to a numbered list

### LINKED ISSUE #230: Numeric affiliations
> Please modify the Affiliations section of your manuscript so that all affiliations listed contain a Department, Institution, City, and Country, and that affiliations are ordered in ascending numerical order. To see a typical title page with formatted affiliations, please see http://journals.plos.org/plosgenetics/s/file?id=RSNU/PLOSGenetics_formatting_sample_title_authors_affiliations.pdf.

We had to do this for deep review.  Manubot creates `author.affiliation_numbers` so do we only need to update the front matter?

### PR DESCRIPTION
**Pull request by dhimmel**
Closes https://github.com/greenelab/meta-review/issues/230

Todo:

- [x] Modify affiliations to include Department, Institution, City, and Country

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -174,8 +174,8 @@ author_info:
     twitter: anthonygitter
     email: gitter@biostat.wisc.edu
     affiliations:
-      - Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison
-      - Morgridge Institute for Research
+      - Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin, United States of America
+      - Morgridge Institute for Research, Madison, Wisconsin, United States of America
```
Comment (dhimmel) [inline content/metadata.yaml:178]: @agitter do you have a department at the "Morgridge Institute for Research"?

**Comment by dhimmel** [comment]
As of https://github.com/greenelab/meta-review/pull/236/commits/1d2af67a79ca1521c9f8bf47d4b1605cdd22c533, looks like:

> image

Ignore the gray rectangle above from [this annoying bug](https://askubuntu.com/q/873813/536520).

File: content/metadata.yaml (hunk 1 above)
Comment (agitter) [reply content/metadata.yaml:178]: Not exactly.  It's best to leave it as you have it now.

**Review by agitter — APPROVED** [review]
This is ready to merge.  As noted elsewhere, we may want to finalize our tagging strategy first.

**Pull request merged**